exchange/bidder.go

```diff
@@ -560,6 +561,7 @@ func (bidder *bidderAdapter) doRequestImpl(ctx context.Context, req *adapters.Re
 		}
 	}
 
+	bidder.me.RecordOverheadTime(metrics.BidderServerResponse, time.Since(httpCallStart))
```

Comment:
Because `bidder.me.RecordOverheadTime(...)` times PBS cpu time, the `overheadTimeBuckets` to log `PreBidder`, `MakeAuctionResponse`, and `MakeBidderRequests` makes perfect sense.  However, in the case of this new metric we expect times to be in the order of the `e.me.RecordAdapterTime(...)` function that records to the `adapterRequestsTimer` prometheus histogram vector based on the `standardTimeBuckets`.

I believe we either should have a different metric that uses the `standardTimeBuckets`:
```
431     metrics.overheadTimer = newHistogramVec(cfg, reg,
432         "overhead_time_seconds",
433         "Seconds to prepare adapter request or resolve adapter response",
434         []string{overheadTypeLabel},
435         overheadTimeBuckets)
436
    +   metrics.bidderServerResponseTimer = newHistogram(cfg, reg,
    +       "bidder-server-response_time",
    +       "Seconds to make a roundtrip to an adapter's bid server",
    +       standardTimeBuckets)
    +   
437     metrics.adapterRequestsTimer = newHistogramVec(cfg, reg,
438         "adapter_request_time_seconds",
439         "Seconds to resolve each successful request labeled by adapter.",
440         []string{adapterLabel},
441         standardTimeBuckets)
metrics/prometheus/prometheus.go
```
Or, a relatively simple option could be to group the "adapter_request_time_seconds" and the new metric under a `map[string]prometheus.HistogramVec` but it will make this an adapter metric, adding a lot of cardinality.
```
431     metrics.overheadTimer = newHistogramVec(cfg, reg,
432         "overhead_time_seconds",
433         "Seconds to prepare adapter request or resolve adapter response",
434         []string{overheadTypeLabel},
435         overheadTimeBuckets)
436
437 -   metrics.adapterRequestsTimer = newHistogramVec(cfg, reg,
438 -       "adapter_request_time_seconds",
439 -       "Seconds to resolve each successful request labeled by adapter.",
440 -       []string{adapterLabel},
441 -       standardTimeBuckets)
    +   m.adapterRequests["roundtrip_only"] = newHistogramVec(cfg, registry,
    +       "adapter_server_rountdtrip_time",
    +       "Seconds to make a roundtrip to an adapter's bid server",
    +       []string{adapterLabel},
    +       standardTimeBuckets)
    +
    +   m.adapterRequests["total_time"] = newHistogramVec(cfg, registry,
    +       "adapter_request_time_seconds",
    +       "Seconds to resolve each successful request labeled by adapter.",
    +       []string{adapterLabel},
    +       standardTimeBuckets)
    +   
442
metrics/prometheus/prometheus.go
```
Thoughts?

Reply:
Thank you for review, Gus! After some discussion with Onkar, we don't want to add an adapter metric, because of high cardinality. Adding a metric that uses the `standardTimeBuckets` is a good option. I am working on the updating the code.

Reply:
This is done in these commits:- [0a6a7ab](https://github.com/prebid/prebid-server/pull/2722/commits/0a6a7ab831df65145042d492d52311222ec3baef), [bb1e5fa](https://github.com/prebid/prebid-server/pull/2722/commits/bb1e5fa1973a7a03432e5e74e1a2ac49458fb2fc), [b445c71](https://github.com/prebid/prebid-server/pull/2722/commits/b445c71e067eabe41c5e2f2e8fb773326a810238), [fd16eac](https://github.com/prebid/prebid-server/pull/2722/commits/fd16eac32b8acb34d3da51cc4019f43367542f71)